src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int shortBase;
+    private int longBase;
+    private int height;
+
+    public IsoscelesTrapezoid(int shortBase, int longBase, int height) {
+        this.shortBase = shortBase;
+        this.longBase = longBase;
+        this.height = height;
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. Also, consider if these values should be generated here or passed from the `FigureSupplier`.